crates/router/src/core/utils.rs

```diff
@@ -130,11 +133,29 @@ pub async fn construct_payout_router_data<'a, F>(
     let payouts = &payout_data.payouts;
     let payout_attempt = &payout_data.payout_attempt;
     let customer_details = &payout_data.customer_details;
+    let connector_label = format!("{}_{}_{}", connector_id, business_country, business_label);
     let connector_customer_id = customer_details
         .as_ref()
         .and_then(|c| c.connector_customer.as_ref())
-        .and_then(|cc| cc.get("id"))
+        .and_then(|cc| cc.get(connector_label))
         .and_then(|id| serde_json::from_value::<String>(id.to_owned()).ok());
+    let vendor_details: Option<PayoutVendorAccountDetails> = match connector_name {
+        api_models::enums::PayoutConnectors::Adyen => None,
+        api_models::enums::PayoutConnectors::Stripe => {
+            payout_data.payouts.metadata.to_owned().and_then(|meta| {
+                let val = meta
+                    .peek()
+                    .to_owned()
+                    .parse_value("PayoutVendorAccountDetails")
+                    .change_context(errors::ApiErrorResponse::MissingRequiredFields {
+                        field_names: ["vendor_details", "individual_details"].to_vec(),
+                    })
```

Comment:
change_context required here?
```suggestion
                    .change_context(errors::ApiErrorResponse::MissingRequiredFields {
                        field_names: ["vendor_details", "individual_details"].to_vec(),
                    })
```

Reply:
Not needed anymore